src/behavioural-events/sdk-events.ts

```diff
@@ -128,6 +130,14 @@ export interface CheckoutPaymentFormImpressionEvent extends ISDKEvent {
   eventName: SDKEventName.CheckoutPaymentFormImpression;
 }
 
+export interface CheckoutPaymentFormErrorEvent extends ISDKEvent {
```

Comment:
Should we get rid of the `CheckoutBillingForm*` events?

Reply:
I guess so 👍🏻